modules/local/dada2_err.nf

```diff
@@ -34,32 +35,32 @@ process DADA2_ERR {
         fnFs <- sort(list.files(".", pattern = "_1.filt.fastq.gz", full.names = TRUE))
         fnRs <- sort(list.files(".", pattern = "_2.filt.fastq.gz", full.names = TRUE))
 
-        sink(file = "${meta.run}.err.log")
+        sink(file = "${prefix}.err.log")
```

Comment:
I'm not sure I understand how this works. I would've guessed that you need also `${meta.run}` or is the run set in the config as `task.ext.prefix` now?

Reply:
Exactely, its now defined in `conf/modules.config` as `ext.prefix = { meta.region ? "region-${meta.region}_run-${meta.run}" : "${meta.run}" }`!